solarmon.cfg.example

```diff
@@ -10,6 +10,7 @@ error_interval = 60
 
 [solarmon]
 port = /dev/ttyUSB0
+inverter_type = SPH-TL3
```

Comment:
Can we leave this as default?
Would probably be a good idea to add some documentation to this file so its easier for people to understand when they first set it up.

```suggestion
# Uncomment to change the model of inverter you are connecting to.
# Possible values: Default, SPH_TL3
inverter_type = Default
```